xknx/devices/binary_sensor.py

```diff
@@ -109,6 +109,11 @@ async def _set_internal_state(self, state):
                 if action.test_if_applicable(self.state, counter):
                     await action.execute()
 
+    @property
+    def counter(self):
```

Comment:
@farmio If we just use the `self._count_set_on` and the set_off properties we would not need to make a new release in order to ship this to HA. What's you opinion on this?